### ISSUE TITLE
License files

### ISSUE THREAD

**Issue by KSXGitHub** [mandreyel/cratetorrent#91]
It is a standard practice to include `LICENSE-MIT` and `LICENSE-APACHE` files at the root of the repo to enable contributors to modify code.

**Comment by mandreyel** [comment]
The license is mentioned in the cargo manifest. Does this suffice or is it still good practice to include license files?

**Comment by nikitalita** [comment]
Yeah, I think so. Github won't show the license for the repo otherwise.

**Comment by CreepySkeleton** [comment]
Also, when a linux distro maintainer packs your crate as `.deb`, his packing system requires these files - `LICENSE-*` - to be present, I think for every crate in workspace, even. They'd sort it out manually one way or another, but having the files present in your repo saves some time of theirs.

This information is half year old and second hand (I'm not a distro maintainer, one of my coworkers is) so I am possibly mistaken, but I bet that including the license files is better than not to from their perspective.
